LAKMobile/src/screens/AddJob.tsx

```diff
@@ -0,0 +1,335 @@
+      </View>
+
+        
+    </TouchableOpacity>
+
+      <View style={styles.photoInstructions}>
+        <AppText style = {styles.photoInstructions}>At least one photo of the package is required.</AppText>
+        <AppText style = {styles.photoInstructions}>Note: The first photo will be the thumbnail of the job listing.</AppText>
+      </View>
+
+      <SingleLineTextInput
+        title = "Job Title"
+        placeholder = "Ex. Box of apples"
+        maxLength={100}
+        onChange={onTextInputChange}
+        value={addJobInformation.jobTitle}
+        id={"jobTitle"}
+        width='60%'
+      />
+
+      <SingleLineTextInput
+        title="Client Name"
+        placeholder="Ex. Gabby Gibson"
+        maxLength={100}
+        onChange={onTextInputChange}
+        value={addJobInformation.clientName}
+        id={"clientName"}
+        width='60%'
+      />
+
+      <SingleLineTextInput
+        title="Date to be delivered"
+        placeholder="Ex. MM/DD/YYYY"
+        footer="(put N/A if not applicable)"
+        maxLength={10}
+        onChange={onTextInputChange}
+        value={addJobInformation.deliveryDate}
+        id={"deliveryDate"}
+        width='60%'
+      />
+
+      <MultilineTextInput
+        title = "Description"
+        placeholder = {"Tell us about your package. Add any extra detail about its size. \n\n Ex. Package will fill up 1/3 of a truck."}
+        maxLength={1000}
+        onChange={onTextInputChange}
+        value={addJobInformation.packageDescription}
+        id={"packageDescription"}
+      />
+      {/*  
+      TODO:  
+      (2) Edit: size of the TextInput box so that it shows all the placeholder text. 
+      Right now you need to scroll down to see all the text.
+       */}
+
+      <SingleLineTextInput
+        title="Package Quantity"
+        placeholder="Ex. 6"
+        maxLength={10}
+        onChange={onTextInputChange}
+        value={addJobInformation.packageQuantity}
+        id={"packageQuantity"}
+        width='40%'
+      />
+
+      {/* TODO: Edit Width */}
+
+      <SingleLineTextInput
+        title="Estimated price of delivery"
+        placeholder="Ex. $$"
+        maxLength={20}
+        onChange={onTextInputChange}
+        value={addJobInformation.estimatedPrice}
+        id={"estimatedPrice"}
+        width='40%'
+      />
+
+      {/* TODO: Edit Width */}
+
+
+      <SingleLineTextInput
+        title="Pick-up location"
+        placeholder="Ex. Insert address or landmark"
+        icon="location-pin"
+        maxLength={100}
+        onChange={onTextInputChange}
+        value={addJobInformation.pickUpLocation}
+        id={"pickUpLocation"}
+        width='80%'
+      />
+
+      <Picker
+        selectedValue={pickupDistrict}
+        onValueChange={(value, index) => setPickupDistrict(value)}
+        mode="dropdown" // Android only
+        style={styles.picker}
+      >
+        {LOCATIONS.map(location => <PickerItem label={location} value={location}/>)}
```

Comment:
This line seems to throw the following warning:
<img width="1016" alt="image" src="https://user-images.githubusercontent.com/56745367/156121981-68ad0704-7837-498b-8623-f8f9d763b9d2.png">
Can add the index of location as key for `PickerItem` for example:
`LOCATIONS.map((location, index) => <PickerItem key={index} label={location} value={location}/>)`